documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -159,8 +161,9 @@ spec:
         namespace: <network_attachment_definition_namespace> <5>
         type: multus
       source:
-        id: <source_network_id>
-        name: <source_network_name>
+        name: <network_attachment_definition> <6>
+        namespace: <network_attachment_definition_namespace> <7>
+        type: multus
```

Comment:
we should say that `type` is an ocp-only field, with a callout, and then it would also be clear that `namespace` is ocp-only by its description below